bucket/joyshockmapper.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.6.0",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
```

Comment:
```suggestion
    "description": "Converts various controllers types input into keyboard and mouse events, allowing them to be used on PC.",
```

Reply:
JoyShockMapper is unique for what it does with gyroscopic aiming and a novel movement scheme the author calls "flick stick". I'll admit that the description I pulled from the source repo isn't the best, but I'm unsure how to word it to include the differentiating features.

I'm happy to use your suggest if you think it fits best with scoop!